cypress/integration/Manual Listing Pages/ManualListingPages.feature

```diff
@@ -0,0 +1,157 @@
+            | desktop           | /about-cancer/treatment/clinical-trials/transitional-cell-cancer-renal-pelvis-ureter | 1,2,Next > |
+
+    Scenario Outline: Meta Tags
+        When user is navigating to "<path>"
+        Then page title is "<page title>"
+        Then the title tag should be "<title tag>"
+        And the page contains meta tags with the following properties
+            | property       | content          |
+            | og:title       | <og:title>       |
+            | og:url         | <og:url>         |
+            | og:description | <og:description> |
+        And the page contains meta tags with the following names
+            | name        | content       |
+            | description | <description> |
+        And there is a canonical link with the href "<canonical link>"
+
+        Examples:
+            | path                                                   | page title                                                             | title tag                                                                                          | og:title                                                               | og:url                                                                         | og:description                                                             | description                                                                | canonical link                                                                 |
+            | /about-cancer/treatment/clinical-trials/covid-19       | NCI-Supported Clinical Trials for Coronavirus                          | NCI-Supported Clinical Trials for Coronavirus - National Cancer Institute                        | NCI-Supported Clinical Trials for Coronavirus                           | https://{CANONICAL_HOST}/about-cancer/treatment/clinical-trials/covid-19 | List of ongoing clinical trials for coronavirus disease 2019 (COVID-19) supported by the National Cancer Institute. Some trials may be specifically for people with cancer who also have coronavirus disease. | List of ongoing clinical trials for coronavirus disease 2019 (COVID-19) supported by the National Cancer Institute. Some trials may be specifically for people with cancer who also have coronavirus disease. | https://{CANONICAL_HOST}/about-cancer/treatment/clinical-trials/covid-19 |
+            | /about-cancer/treatment/clinical-trials/cam-procedures  | Clinical Trials for Complementary or Alternative Medicine Procedure(s)| Clinical Trials for Complementary or Alternative Medicine Procedure(s) - National Cancer Institute | Clinical Trials for Complementary or Alternative Medicine Procedure(s) | https://{CANONICAL_HOST}/about-cancer/treatment/clinical-trials/cam-procedures | Find clinical trials for complementary or alternative medicine procedures. | Find clinical trials for complementary or alternative medicine procedures.| https://{CANONICAL_HOST}/about-cancer/treatment/clinical-trials/cam-procedures |
+    
```

Comment:
we want to test meta tags  for all url - please add info for remaining 11  pages (take it from prod)